src/modules/vtol_att_control/tiltrotor.cpp

```diff
@@ -190,27 +201,21 @@ void Tiltrotor::update_mc_state()
 		set_idle_mc();
 		flag_idle_mc = true;
 	}
-}
 
-void Tiltrotor::process_mc_data()
-{
-	fill_att_control_output();
+	// set the mc attitude control weights to one
```

Comment:
You can probably drop this comment. It doesn't really add too much value over the code below it, unless you'd rather state why you set it to one ;)